src/Workspaces/CSharp/Portable/Utilities/CSharpUseExplicitTypeHelper.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle.TypeStyle;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Utilities
+{
+    internal sealed class CSharpUseExplicitTypeHelper : CSharpTypeStyleHelper
```

Comment:
This was moved down from CSharpFeatures.  Other than renaming it, it was not changed.